src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetExchanger.java

```diff
@@ -163,14 +133,14 @@ public void toggleMode(EntityPlayer player, ItemStack heldItem) {//TODO unused
 
     public void setMode(EntityPlayer player, ItemStack heldItem, int modeInt) {
         //Called when we specify a mode with the radial menu
```

Comment:
Requires a space

Reply:
Well, I have never touched this line in the PR, and there are comments like this all over the place. I think they should have a dedicated commit/PR.